ci-operator/config/openshift/jobset-operator/openshift-jobset-operator-main.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  ocp_builder_rhel-9-base-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-base-openshift-4.19
```

Comment:
@ardaguclu I took this image from your PR. It seems okay to me, but cannot track any other usages or recommendations for this image. Do you have more details on this?

Reply:
That is a nice catch. Base image should be independent from the OCP version. So I opened these;

* https://github.com/openshift/lws-operator/pull/11
* https://github.com/openshift/release/pull/61493

Basically, `base-rhel9` would better align with other components.

Reply:
sgtm, changed as well